From 7f0aa58747b7335c7856fb7153be6cd1e22b74b6 Mon Sep 17 00:00:00 2001
From: Michal Schmidt <mschmidt@redhat.com>
Date: Fri, 2 Dec 2011 11:32:52 +0100
Subject: [PATCH] rc-local: order after network.target

As suggested by Bill Nottingham: rc.local is often used for frobbing the
network.

https://bugzilla.redhat.com/show_bug.cgi?id=754789
(cherry picked from commit 91b684c7300879a8d2006038f7d9185d92c3c3bf)
---
 units/fedora/rc-local.service |    1 +
 units/suse/rc-local.service   |    1 +
 2 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/units/fedora/rc-local.service b/units/fedora/rc-local.service
index 36100df..0bef5c7 100644
--- a/units/fedora/rc-local.service
+++ b/units/fedora/rc-local.service
@@ -9,6 +9,7 @@
 # systemd-rc-local-generator if /etc/rc.d/rc.local is executable.
 [Unit]
 Description=/etc/rc.d/rc.local Compatibility
+After=network.target
 
 [Service]
 Type=forking
diff --git a/units/suse/rc-local.service b/units/suse/rc-local.service
index 11dc581..454095c 100644
--- a/units/suse/rc-local.service
+++ b/units/suse/rc-local.service
@@ -9,6 +9,7 @@
 # systemd-rc-local-generator if /etc/init.d/boot.local is executable.
 [Unit]
 Description=/etc/init.d/boot.local Compatibility
+After=network.target
 
 [Service]
 Type=oneshot
